### PR TITLE
Add pfcat, pfdel scripts from cfengine

### PR DESCRIPTION
**Pull request by detjensrobert**
As requested in osuosl-cookbooks/osl-mailrelay#4.

### CODE REVIEW

File: files/server/pfcat [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	if [ -n "$FILE" ]; then
+		postcat $FILE
+		exit 0
+	fi
+done
+
+if [ -z $FILE ]; then
+	echo "No such queue file $1"
+	exit 1
+fi
```
Comment (ramereth) [inline files/server/pfcat:39]: Can you please run `shellcheck` on this and fix the issues? Also please convert all tabs to spaces.

File: files/server/pfdel [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+my $POSTSUPER = "/usr/sbin/postsuper";
+
+my $email_addr = "";
+my $qid = "";
+my $euid = $>;
+
+if ( @ARGV !=  1 ) {
+	die "Usage: pfdel <email_address>\n";
+} else {
+	$email_addr = $ARGV[0];
```
Comment (ramereth) [inline files/server/pfdel:22]: Can you please convert all tabs to spaces and also convert this to two space indents like we do with ruby code?

File: test/integration/server/inspec/server_spec.rb [hunk 3]
```diff
@@ -5,3 +5,27 @@
...
+describe command '/usr/local/bin/pfcat' do
+  its('stdout') { should match /Usage: pfcat <queue id>/ }
+end
+
+describe file '/usr/local/bin/pfdel' do
+  it { should exist }
+  it { should be_executable }
+  its('content') { should match /# pfdel - Deletes message containing specified address from the Postfix queue./ }
+end
+
```
Comment (ramereth) [inline test/integration/server/inspec/server_spec.rb:28]: I recommend you add the following tests as well (note: I'm not including the full path):
```
# should return 1
pfcat foo
# should return 255
pfdel foo@bar.com
```

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by ramereth — APPROVED** [review]
Please do a minor bump.

**Comment by detjensrobert** [comment]
!bump minor ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v1.3.0.  Have a nice day!

**Pull request merged**